src/plugins/kibana_overview/public/components/app.tsx

```diff
@@ -49,7 +52,12 @@ export const KibanaOverviewApp = ({
       <I18nProvider>
         <Switch>
           <Route exact path="/">
-            <Overview newsFetchResult={newsFetchResult} solutions={solutions} features={features} />
+            <Overview
+              newsFetchResult={newsFetchResult}
+              solutions={solutions}
+              features={features}
+              dataViewEditor={dataViewEditor}
```

Comment:
Same question as above: one of the reason we use a context is to avoid prop drilling. Why not access it inside of `<Overview />`?